.coderabbit.yaml

```
@@ -0,0 +1,67 @@
+    - "README.md"
+    - "**/README.md"
+  exclude_paths:
+    - "**/node_modules/**"
+    - "**/__pycache__/**"
+    - "**/dist/**"
+    - "**/build/**"
+
+# Code generation settings
+code_generation:
+  enabled: true
+  docstrings: true 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)